frontend/tests/controllers-test/challengeCtrl.test.js

```
@@ -2846,8 +2839,7 @@ describe('Unit tests for challenge controller', function () {
             var editChallengePhaseForm = false;
             success = false;
             vm.editChallengePhase(editChallengePhaseForm);
-            expect(utilities.storeData).toHaveBeenCalledWith('emailError', errorResponse.detail);
-            expect($state.go).toHaveBeenCalledWith('web.permission-denied');
+            expect(utilities.handlePermissionDeniedError).toHaveBeenCalled();
```

ESLint

[error] 'expect' is not defined.

(no-undef)